src/commands/init.ts

```diff
@@ -80,59 +66,26 @@ export class InitCommand implements Command {
           } else {
             logger.warn(`Template ${options.name} not found`);
             logger.debug(`Yeoman generator ${generatorName} not found`);
+            reject(`Template ${options.name} not found`);
```

Comment:
Yeah, that was in the original code. I was a bit confused by it, but I left it in. I'll go ahead and take it out.